lib/tasks/assemble.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
 const chalk = require('chalk');
-const install = require('rsvp').denodeify(require('npmi'));
+const pruner = require('rsvp').denodeify(require('electron-packager/prune').pruneModules);
```

Comment:
I don't think this is really a public API, so I'm kinda uncomfortable using it like this. My preference would be to copy the code that we need into this project, but failing that, at the very least list `electron-packager` as one of our dependencies, rather than just implicitly relying on `electron-forge` to make it available to us.